src/test_workflow/integ_test/integ_test_suite_opensearch_dashboards.py

```diff
@@ -60,7 +60,7 @@ def __init__(
 
         self.repo = GitRepository(
             build_manifest_opensearch_dashboards.components['functionalTestDashboards'].repository,
-            build_manifest_opensearch_dashboards.components['functionalTestDashboards'].commit_id,
+            build_manifest_opensearch_dashboards.components['functionalTestDashboards'].ref,
```

Comment:
Want to add a simple test verifying that ref is checked out and not commit_id?

Reply:
Updated, thanks.